packages/web-components/src/accordion/accordion-item/accordion-item.ts

```diff
@@ -0,0 +1,42 @@
+import { attr } from '@microsoft/fast-element';
+import { FASTAccordionItem } from '@microsoft/fast-foundation';
+import { AccordionItemExpandIconPosition, AccordionItemSize } from './accordion-item.options.js';
+
+/**
+ * @internal
+ */
+class AccordionItem extends FASTAccordionItem {
+  /**
+   * Defines accordion header size.
+   *
+   * @public
+   * @default 'medium'
+   * @remarks
+   * HTML Attribute: appearance
+   */
+  @attr({ attribute: 'size' })
+  public size: AccordionItemSize;
+
+  /**
+   * Sets the width of the focus state.
+   *
+   * @public
+   * @remarks
+   * HTML Attribute: block
+   */
+  @attr({ mode: 'boolean' })
+  block: boolean = true;
+
+  /**
+   * Sets expand and collapsed icon position.
+   *
+   * @public
+   * @default 'start'
+   * @remarks
+   * HTML Attribute: expandIconPosition
+   */
+  @attr({ attribute: 'expandIconPosition' })
```

Comment:
Attribute syntax please :)

```suggestion
  @attr({ attribute: 'expand-icon-position' })
```